scripts/run-qemu-ota

```diff
@@ -2,126 +2,12 @@
 
 from argparse import ArgumentParser
 from subprocess import Popen
-from os.path import exists, join, realpath
-from os import listdir
-import random
+from os.path import exists, join
 import sys
-import socket
+import QemuCommand
```

Comment:
This refactor is great, but one minor nit: can you use the [pep8](https://www.python.org/dev/peps/pep-0008/#package-and-module-names) convention for the name of the QemuCommand module, please?  e.g:

     from qemucommand import QemuCommand